playbooks/tasks/portal-docker-services-start.yml

```diff
@@ -50,25 +50,35 @@
   debug:
     msg: "{{ inventory_hostname + ' docker services start: ' + lookup('pipe','date +%Y-%m-%dT%H:%M:%S') + ' UTC' }}"
 
-# Start the docker services (on deploy also build images)
-- name: Start the docker services (on deploy also build images)
-  community.docker.docker_compose:
-    project_src: "{{ webportal_dir }}"
-    files: "{{ webportal_docker_compose_files_wanted }}"
-    build: "{{ docker_compose_build }}"
-    remove_orphans: True
-    nocache: True
-    pull: True
-    state: present
-    # TODO: Build in parallel, when Ansible support is added
+- name: Pull docker images
+  ansible.builtin.command: "chdir={{ webportal_dir }} ./dc pull"
   become: True
-  become_user: "user"
-  register: docker_compose_start_result
+  become_user: "{{ webportal_user }}"
+
+- name: Build docker images
```

Comment:
Is this a no-op if the images are pulled from docker hub?

Reply:
yes but will be used when deploying a custom branch (for example for skyd)